pkg/cmd/cli/cmd/set/env.go

```diff
@@ -273,14 +382,45 @@ func RunEnv(f *clientcmd.Factory, in io.Reader, out io.Writer, cmd *cobra.Comman
 				c.Env = updateEnv(c.Env, env, remove)
 
 				if list {
+					resolveErrors := map[string][]string{}
+					store := newResourceStore()
+
 					fmt.Fprintf(out, "# %s %s, container %s\n", info.Mapping.Resource, info.Name, c.Name)
```

Comment:
presumably this is formatted this way so that the output of a list can be used in a shell script to set the envvars? if they ask us to resolve, and we can't, we should probably ensure we have a non-zero exit code